jenkins-scripts/docker/ign_cmake-compilation.bash

```diff
@@ -31,6 +31,6 @@ if ! [[ ${IGN_CMAKE_MAJOR_VERSION} =~ ^-?[0-9]+$ ]]; then
   exit -1
 fi
 
-export GZDEV_PROJECT_NAME="ignition-cmake${IGN_CMAKE_MAJOR_VERSION}"
+export GZDEV_PROJECT_NAME="gz-cmake${IGN_CMAKE_MAJOR_VERSION}"
```

Comment:
btw, these changes don't need to wait for the CMake project names to change, because we already have aliases on gzdev:

* https://github.com/gazebo-tooling/gzdev/pull/57